src/patterns/technical-problem-pages/link-to-another-service/index.njk

```diff
@@ -0,0 +1,20 @@
+---
+layout: layout-example.njk
+---
+
+<div class="govuk-width-container">
+
+  <div class="govuk-main-wrapper">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+
+      <h1 class="govuk-heading-xl">Sorry, there is a problem with the service</h1>
+      <p class="govuk-body-l">Try again later.</p>
+      <p class="govuk-body">If you need to make a payment and know how much you need to pay, <a class="govuk-link" href="#">use the payment service</a>.</p>
```

Comment:
Change the line to:

`<p class="govuk-body">You can <a class="govuk-link" href="#">change other VAT details</a>.</p>`